GoogleSignInSwift.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |s|
+  s.name = 'GoogleSignInSwift'
+  s.version = '0.1.0'
```

Comment:
I think we should sync versions with the main pod, setting this to 6.2.0.  Although we could have independent versioning for the pods, we couldn't do this for the SPM products because SPM uses one git tag to version all products in the package definition.

Reply:
Okay, I'll set this to 6.2.0. One great leap forward!